### PR TITLE
A small patch for the rebasing of the main directory under src

### LINKED ISSUE #63: Restructuring this package to fit newer python packaging standard
I think it may be valuable to restructure this package to [a newer standard](https://packaging.python.org/en/latest/tutorials/packaging-projects/).
It should be relatively easy to do and will establish a "cleaner" structure.

Using [some cookie-cutter template](https://github.com/ionelmc/cookiecutter-pylibrary) may do the trick, but I would love to get your input on this suggestion before I start working on it :satisfied:

Thanks in advance!

### PR DESCRIPTION
**Pull request by GalBenZvi**
Just a small fix - updated the location of the `._version.py` written by `setup_scm`. Fixes #63

### CODE REVIEW

**Comment by arokem** [comment]
Test data now lives here: https://zenodo.org/record/6721110#.YrXPyRPMLKp

**Comment by arokem** [comment]
The tests on this branch are still pretty hopelessly broken. I tried to work towards fixing them in https://github.com/GalBenZvi/eddymotion/pull/3, but there are still failures I don't understand. I would need @oesteban to explain to me how the logo implementation works before I can make progress. In particular, how does the model `fit` method know that only some of the data is provided as training data (e.g., [on this line](https://github.com/nipreps/eddymotion/blob/main/src/eddymotion/tests/test_model.py#L107)) and to account for that fact when selecting which volumes conform with b0 requirements that are set (for example) on [this line](https://github.com/nipreps/eddymotion/blob/main/src/eddymotion/model.py#L276)?

File: .gitignore [hunk 1]
```diff
@@ -113,3 +113,6 @@ ENV/
 
 #kubernetes stuff
 kubernetes/jobs/
+
+# Version files
+*_version.py
```
Comment (oesteban) [inline .gitignore:118]: Better be specific, right?
```suggestion
src/eddymotion/_version.py
```

**Review by oesteban — APPROVED** [review]

**Comment by oesteban** [comment]
@arokem It seems we should work on the documentation actually - if you can't wrap your head around how it works then I'm pretty positive that literally any other outsider will fall into that trap. I'm opening #76 for this.

**Comment by oesteban** [comment]
Check out #84. In particular, there was a bug using the gradient table of the full dataset instead of that of the training subsample (i.e., N - 1).

**Comment by arokem** [comment]
Thanks! That helps! See questions on #84 for followup.

**Pull request merged**